test/test-mda-models/aissemble-test-data-delivery-pyspark-model-basic/pyproject.toml

```diff
@@ -22,7 +22,7 @@ aissemble-extensions-encryption-vault-python = {path = "../../../extensions/exte
 kafka-python = "^2.0.2"
 python = ">=3.8"
 krausening = ">=20"
-pyspark = "3.4.0"
+pyspark = "3.5.2"
```

Comment:
Q/S: Do we need to update any the pyproject.toml files for data-records/pipelines in the MDA templates or is that already driven by the POM property? (I know Habushu would automatically update them via `managedDependencies`, but it's cleaner if they come out right from MDA generation.)

Reply:
The pyproject.toml vm explicitly sets the pyspark version using its configs that are set by pom properties. Same way its handling aissemblePythonVersion or tool.poetry name
`pyspark = "${versionSpark}"`